stories/Atom/Reach-element/Table/table.scss

```diff
@@ -4,29 +4,129 @@
 
 /* table tag start */
 table {
-    border-spacing: 0px;
+  border-collapse: collapse;
+  border-spacing: 0px;
+  width: 100%;
+  @include devicebreak(medium) {
+    width: auto;
+  }
+
+  &.small {
+    thead,
+    tbody {
+      tr {
+        th,
+        td {
+          padding: 12px 40px 12px 12px;
+          font-size: $font-size-16;
+          line-height: 22px;
+        }
+      }
+    }
+  }
+
+  &.striped {
+    tr {
+      &:nth-child(even) {
+        background-color: $color-gray-200;
+      }
+      @extend %border-none;
+    }
+  }
+
+  &.border {
+    border-collapse: collapse;
+    tr {
+      th,
+      td {
+        border: 1px solid $color-gray-400;
+      }
+    }
+  }
+
+  &.stacked {
+    tr {
+      th,
+      td {
+        display: block;
+        @include devicebreak(medium) {
+          display: table-cell;
+        }
+      }
+    }
+    thead {
+      tr {
+        display: none;
+        @include devicebreak(medium) {
+          display: table-row;
+        }
+      }
+    }
+    &.border {
+      tr {
+        border: 1px solid $color-gray-400;
+        th,
+        td {
+          @extend %border-none;
+          @include devicebreak(medium) {
+            border: 1px solid $color-gray-400;
+          }
+        }
+      }
+    }
+  }
+}
+
+thead,
+tbody {
   tr {
-    th {
+    th,
+    td {
       vertical-align: top;
-      font-family: $Font-Family-ProximaRegular;
-      padding: 12px;
-      color: $color-black;
-      background-color: #edeff0;
+      padding: 12px 40px 12px 12px;
+      font-size: $font-size-20;
+      line-height: 28px;
+    }
+  }
+}
+
+thead {
+  tr {
+    th {
+      font-family: $Font-Family-ProximaSemiBold;
+      background-color: $color-gray-300;
       text-align: left;
-      font-weight: 700;
-      @extend %paragragh-font-20;
-      @include devicebreak(medium) {
-        padding: 16px;
+    }
+  }
+}
+
+tbody {
+  tr {
+    @extend %border-bottom;
+  }
+}
+
+.scroll {
+  overflow-x: scroll;
+  @include devicebreak(medium) {
+    overflow-x: inherit;
+  }
+}
+
+[dir="rtl"] {
+  table {
+    thead {
+      tr {
+        th {
+          text-align: right;
+        }
       }
     }
-    td {
-      vertical-align: top;
-      padding: 12px;
-      @extend %border-bottom;
-      font-weight: 300;
-      @extend %paragragh-font-20;
-       @include devicebreak(medium) {
-        padding: 16px;
+    tbody {
+      tr {
+        td {
+          text-align: right;
+        }
```

Comment:
@harishkumar-srijan can we avoid nesting in css file and we can just keep like tr ,th or td and only wrap in tbody , thead or tr when it is required

Reply:
done, remove all nested CSS in the file and keep it where it's needed.